tests/integration/it_utils.py

```diff
@@ -119,8 +122,13 @@ async def fund_wallet_async(
         destination=wallet.address,
         amount=FUNDING_AMOUNT,
     )
-    await sign_and_submit_async(payment, client, MASTER_WALLET, check_fee=True)
-    await client.request(LEDGER_ACCEPT_REQUEST)
+
+    # concurrent wallet_fund operations will attempt to advance the ledger at the same
+    # time. Consequently, all the funding operations fail, except for the first one.
+    # using a lock will serialize the access to this critical operation
+    async with async_wallet_fund_lock:
```

Comment:
@mvadari  I found that adding a lock to this section of the code eliminated the "Account Not Found" error.